### DISCUSSION TITLE
Need details for a too long evaluating submission!

### DISCUSSION THREAD

**Discussion by bigoood** [PaddlePaddle/PaddleSpatial#258] [BaiduKDDCup2022] [answered]
Hi, team name : meatTeam; submitted time: 2022-07-12 18:31. And now it is still in evaluating, can you give me some details about it?

**Comment by ramber1836** [comment, marked as answer]
The log is not available when the evaluation is running.

**Reply by bigoood** [reply]
> The log is not available when the evaluation is running.

is there no timeout for this too long evaluating?

**Reply by ramber1836** [reply]
10 hours

**Reply by bigoood** [reply]
![1](https://user-images.githubusercontent.com/109197270/178728243-df872581-b840-44e3-b0c6-bacd55972593.png)
@ramber1836 Maybe, the time budget has been changed to 20h? And will not be shutdown in advance until consuming 20h?

**Reply by ramber1836** [reply]
The timeout is configured as 10 hours in the evaluation code: 
<img width="796" alt="image" src="https://user-images.githubusercontent.com/19561583/178736544-e3838486-a38c-49b2-a4fb-e1d7e588c55c.png">

A system timeout is also configured, which is 20 hours. 
Most of the time, the first timeout violation will be checked by default (after 2 iterations), but if your code consumes too much time, like, cannot be finished within two runs in 20 hours, then the system timeout will be triggered.
